Makefile

```diff
@@ -17,8 +17,9 @@ lint: ## Run the linter
 test_examples: ## Run tests for the examples
 	@mkdir -p $(EXAMPLES_TARGET)
 	@for dir in $(EXAMPLES_PATHS); do \
-		echo "Processing $$dir" ; \
-		cargo test --quiet --manifest-path $$dir/Cargo.toml --release --target-dir $(EXAMPLES_TARGET); \
+		if [ $$dir = $(EXAMPLES_TARGET) ]; then continue; fi; \
```

Comment:
Can you check whether changing `EXAMPLES_PATHS` to sth like
```
EXAMPLES_PATHS := $(shell find $(EXAMPLES)-mindepth 1 -maxdepth 1 -name Cargo.toml | xargs dirname)
```
help here? i.e. allow for getting rid of this little hack

Reply:
had to change it a bit, but in the end hack was removed :)